docs/builtin-functions.md

```diff
@@ -45,7 +45,7 @@ Python contains a limited number of builtin functions defined [here](https://doc
 | *`list`* | implemented as a tuple |
 | `locals` | No |
 | **`map`** | as a loop iterable |
-| **`max`** | Fortran-only |
+| **`max`** | Yes |
 | `memoryview` | No |
 | **`min`** | Fortran-only |
```

Comment:
Two comments here:
1. This is a list of Python built-in functions, not Numpy functions. The title and description of this PR refer explicitly to `numpy.max`, not the built-in Python `max`. Are they both supported now?
2. Is there a reason for not adding support to `min` in the same PR?

Reply:
My Bad, i didn't add support for built in python max. I will change it Back to fortran only

Reply:
I have created a new pr for min since i was solving the issue1621 in this PR.
If you say i will add numpy.min in this PR

Reply:
As the code for `max` and `min` should be almost identical, it feels natural to tackle them both in the same PR. This also makes the review faster.

In this case, since you have already created a separate PR for `min`, you can leave things as they are.

Reply:
ok sir then i will run the tests and then you can check again